src/electron/platform/android/test-configs/automated-checks/test-config.tsx

```diff
@@ -22,5 +25,11 @@ export const automatedChecksTestConfig: TestConfig = {
         resultsFilter: automatedChecksResultsFilter,
         allowsExportReport: true,
         visualHelperSection: ScreenshotView,
+        startOverButtonTabSettings(props: ReflowCommandBarProps): StartOverButtonTabSettings {
```

Comment:
Currently, the `resultsFilter` values for automated checks and needs review live separately from their respective test configs and have their own test files. That way, the test configs contain no logic, just keys and values, and don't need tests. Do we want to follow a similar pattern for these `startOverButtonTabSettings`?

Reply:
Good call. I created a function that gets shared in AutomatedChecks and NeedsReview, and another function (just a placeholder until we have the connecting code) for TabStops. These functions have unit tests to replace the ones that were removed from the ReflowCommandBar class